Services/OrderService.js

```diff
@@ -2,7 +2,8 @@ let createNewOrder = async (db, order) => {
     const collection = db.collection('orders')
     const result = await collection.insertOne({
         name: order.name,
-        deliveryAddress: order.deliveryAddress,
+        firstLineOfAddress: order.firstLineOfAddress,
+        postcode: order.postcode,
         email: order.email,
     })
```

Comment:
Because you have already validated the order object, you could just do .insertOne(order) instead to have the same effect

Reply:
Your validation is robust enough to do this without worrying, since it does not allow any extra properties to be in the object you don't have to worry about someone inserting something random